doc/contributing/building-node-with-ninja.md

```diff
@@ -38,6 +38,9 @@ make -j4 # With this flag, Ninja will limit itself to 4 parallel jobs,
          # regardless of the number of cores on the current machine.
 ```
 
+Note: if you are on macOS and use GNU Make version `v3.8.x`, the `-jn` flag
```

Comment:
It's `3.81` (not `3.8.1`), but I think it's fine to broaden that to `3.x`:

```suggestion
Note: if you are on macOS and use GNU Make version `3.x`, the `-jn` flag
```

Also, the `v` is redundant since you explicitly wrote `version` before that.